### ISSUE TITLE
Issues with plotting mgmwm result

### ISSUE THREAD

**Issue by stefangachter** [SMAC-Group/mgmwm#4]
I used to plot `mgmwm` outcome with `plot` as follows
```
estimModel = mgmwm(axisImuObjs, model, CI=TRUE)
plot(estimModel, decomp=TRUE)
```
but for some reason, I obtain the error:
```
 xy.coords(x, y, xlabel, ylabel, log) でエラー: 
  'x' is a list, but does not have components 'x' and 'y'
```
and I have to resort to
```
plot.mgmwm(estimModel, decomp=TRUE)
```
Somehow plot does not recognize `mgmwm` object anymore, but I don't understand why. Any suggestion?

**Comment by stefangachter** [comment]
Here, an example:
```
library(imudata)
library(mgmwm)

axisImuData <- list()

data(adis_16405_imu1)
axisImuData[length(axisImuData)+1] <- list(as.vector(adis_16405_imu1[,1]))

data(adis_16405_imu2)
axisImuData[length(axisImuData)+1] <- list(as.vector(adis_16405_imu2[,1]))

titleLabel = 'ADIS 16405 IMU1 and IMU2'
xLabel = 'Time Scale (s)'
yLabel = 'Variance ((deg/s)^2)'
axisLabel = 'Gyro_X'
expLabels = c('IMU1', 'IMU2')

axisImuObjs <- do.call(make_mimu, c(axisImuData, list(freq=100, unit='s', sensor.name=paste(titleLabel, ', ', axisLabel, sep=''), exp.name=paste(expLabels, sep =''))))

plot(axisImuObjs, xlab=xLabel, ylab=yLabel)

model <- AR1() + RW()
estimModel = mgmwm(axisImuObjs, model, CI=TRUE)
print(summary(estimModel))

plot(estimModel, decomp=TRUE, ylab_mgmwm=yLabel)
```

**Comment by stefangachter** [comment]
And here, the versions:
```
R version 4.0.5 (2021-03-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Japanese_Japan.932  LC_CTYPE=Japanese_Japan.932    LC_MONETARY=Japanese_Japan.932
[4] LC_NUMERIC=C                   LC_TIME=Japanese_Japan.932    
system code page: 1252

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] mgmwm_0.1.0      openxlsx_4.2.3   gmwm_3.0.0       ggplot2_3.3.3    imudata_0.1.9    assertthat_0.2.1

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.6         lattice_0.20-41    tidyr_1.1.3        prettyunits_1.1.1  ps_1.6.0           rprojroot_2.0.2   
 [7] utf8_1.2.1         gmp_0.6-2          R6_2.5.0           plyr_1.8.6         backports_1.2.1    coda_0.19-4       
[13] pillar_1.6.0       iterpc_0.4.2       rlang_0.4.11       progress_1.2.2     curl_4.3.1         rstudioapi_0.13   
[19] callr_3.7.0        robcor_0.1-6       desc_1.3.0         devtools_2.4.1     stringr_1.4.0      munsell_0.5.0     
[25] broom_0.7.6        compiler_4.0.5     pkgconfig_2.0.3    pkgbuild_1.2.0     arrangements_1.1.9 tidyselect_1.1.1  
[31] tibble_3.1.1       fansi_0.4.2        crayon_1.4.1       dplyr_1.0.5        withr_2.4.2        grid_4.0.5        
[37] gtable_0.3.0       lifecycle_1.0.0    magrittr_2.0.1     scales_1.1.1       zip_2.1.1          cli_2.5.0         
[43] stringi_1.5.3      cachem_1.0.4       farver_2.1.0       reshape2_1.4.4     fs_1.5.0           remotes_2.3.0     
[49] testthat_3.0.2     wv_0.1.1           ellipsis_0.3.2     generics_0.1.0     vctrs_0.3.8        tools_4.0.5       
[55] glue_1.4.2         purrr_0.3.4        hms_1.0.0          processx_3.5.2     pkgload_1.2.1      fastmap_1.1.0     
[61] colorspace_2.0-1   sessioninfo_1.1.1  memoise_2.0.0      usethis_2.0.1      simts_0.1.1  
```

**Comment by stefangachter** [comment]
In the above example, first 'plot` works:
![image](https://user-images.githubusercontent.com/7974580/122000918-d9ae9400-cdaf-11eb-8ac3-a635733a669a.png)
but after model parameter estimation, summary is wrong:
```
Length Class    Mode     
estimates       3     -none-   numeric  
obj_value       1     -none-   numeric  
model_hat      11     ts.model list     
scales_max_vec 18     -none-   numeric  
p_value         1     -none-   logical  
test_result     1     -none-   character
distrib_param   9     -none-   numeric  
ci_low          3     -none-   numeric  
ci_high         3     -none-   numeric  
mimu            2     mimu     list  
```
Summary should show the model parameters and the second `plot` does not plot the wavelet variances and the models. Instead, it throws the error:
```
 xy.coords(x, y, xlabel, ylabel, log) でエラー: 
  'x' is a list, but does not have components 'x' and 'y'
```

**Comment by stefangachter** [comment]
Problems appear with R 4.0.5 but not with R 3.6.3

**Comment by lionelvoirol** [comment]
Hi Stefan,

Following your reproducible example, I've identified the little bug with the S3 method `plot.mgmwm` and `summary.mgmwm`.

I just tried your example and everything seems to work fine now when calling `summary(estimated_model)` or `plot(estimated_model)` where `estimated_model` is a `mgmwm` S3 object.

Let me know if it also works on your side after updating `mgmwm` from GitHub.

Best

Lionel
